src/averageAges.js

```diff
@@ -15,11 +15,20 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menList = people.filter(user => user.sex === 'm');
+
+  menList = century
```

Comment:
It is better use more obvoius check here, because century, theoretically, can have falsy value (0)
```suggestion
  menList = century === undefined
```